ReactCommon/react/renderer/core/ShadowNode.cpp

```diff
@@ -213,7 +213,7 @@ void ShadowNode::replaceChild(
       *std::const_pointer_cast<ShadowNode::ListOfShared>(children_);
   auto size = children.size();
 
-  if (suggestedIndex != -1 && suggestedIndex < size) {
+  if (suggestedIndex != -1 && static_cast<size_t>(suggestedIndex) < size) {
```

Comment:
should this be >=0 instead now?

Reply:
If `suggestedIndex` is actually negative, it'll hit an out of range exception on the next line anyway.  So its already being checked to some degree.